src/dev_genesis_wasm.erl

```diff
@@ -244,5 +244,194 @@ log_server_events(Bin) when is_binary(Bin) ->
     log_server_events(binary:split(Bin, <<"\n">>, [global]));
 log_server_events([Remaining]) -> Remaining;
 log_server_events([Line | Rest]) ->
-    ?event(genesis_wasm_server, {server_logged, Line}),
-    log_server_events(Rest).
\ No newline at end of file
+    ?event(genesis_wasm_server, {server_logged, {string, Line}}),
+    log_server_events(Rest).
+
+%% Tests
+-ifdef(ENABLE_GENESIS_WASM).
+
+genisis_wasm_opts() ->
```

Comment:
@jfrain99 might be good to add some documentation describing some of the core functions

```
%% @doc Generate a process message with a random number, and no 
%% executor.

```

see dev_process for some examples